src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Ball {
+
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public String getColor() {
+        return this.color;
+    }
+
+    public int getNumber() {
+        return this.number;
+    }
+
```

Comment:
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#please-dont-add-redundant-empty-lines-to-your-code